ark/utils/test_utils.py

```diff
@@ -564,3 +565,78 @@ def create_test_extraction_data():
     channel_data[:, 15:25, 20:30, 4] = 10
 
     return cell_mask, channel_data
+
+
+def save_figure(save_dir, save_file):
+    """Verify save_dir and save_file, then save to specified location
+
+    Args:
+        save_dir (str):
+            the name of the directory we wish to save to
+        save_file (str):
+            the name of the file we wish to save to
+    """
+
+    # verify save_dir exists
+    if not os.path.exists(save_dir):
+        raise FileNotFoundError("save_dir %s does not exist" % save_dir)
+
+    # verify that if save_dir specified, save_file must also be specified
+    if save_file is None:
+        raise FileNotFoundError("save_dir specified but no save_file specified")
+
+    plt.savefig(os.path.join(save_dir, save_file))
+
+
+def get_fov_info(all_data, fov_col, included_fovs=None):
+    """Generate the included_fovs and the number of fovs
+
+    Args:
+        all_data (pandas.DataFrame):
+            data including fovs, cell labels, and cell expression matrix for all markers
+        fov_col (str):
+            the name of the column in all_data identifying the fovs
+        included_fovs (list):
+            patient labels to include in analysis. If argument is none, default is all labels used.
+
+    Returns:
+        tuple (list, int):
+
+        - a list of the fovs to use
+        - the number of fovs
+    """
+
+    if included_fovs is None:
+        included_fovs = all_data[fov_col].unique()
+        num_fovs = len(included_fovs)
+    else:
+        num_fovs = len(included_fovs)
+
+    return included_fovs, num_fovs
+
+
+def verify_spatial_info(all_data, fov_col, included_fovs, excluded_colnames=None):
```

Comment:
The reason I described this in more general terms in my comment was because there are other places in the repo where we do this. 
For example [here](https://github.com/angelolab/ark-analysis/blob/cd51002e622c802fc44114ac6dd3431edf806682/ark/segmentation/marker_quantification.py#L259]). 

We can turn this into a function that checks if membership of one list with another, and reports errors according to whether all of list A have to be in list B, or only if parts of list A aren't in list B, depending on the context.